src/css/custom.css

```diff
@@ -18,7 +18,7 @@
   --ifm-color-primary-lighter: #377fad;
   --ifm-color-primary-lightest: #408fc2;
   --ifm-code-font-size: 85%;
-  --ifm-font-size-base: 1rem;
+  --ifm-font-size-base: 0.85rem;
```

Comment:
Not sure why it's back to 0.85 here, maybe no fetch + rebase before PR. Just set it back to 1rem